src/SFA.DAS.RoATPService.Database/Script.PostDeployment1.sql

```diff
@@ -355,6 +355,9 @@ BEGIN
 	SET IDENTITY_INSERT dbo.[ProviderTypeOrganisationStatus] OFF
 END
 
+-- APR-474 update any non-set SourceIsUKRLP to true for Organisations.OrganisationData
+update organisations set OrganisationData = JSON_Modify(OrganisationData,'$.SourceIsUKRLP',CAST(1 as BIT)) where JSON_VALUE(OrganisationData,'$.SourceIsUKRLP') is null
```

Comment:
We set boolean values in other organisation data fields so best to be consistent?